test/unit/menu/MenuList.test.tsx

```diff
@@ -3,7 +3,7 @@ import {assert} from 'chai';
 import * as td from 'testdouble';
 import {shallow, mount} from 'enzyme';
 import {MenuList} from '../../../packages/menu/index';
-import List from '../../../packages/list/index';
+import List from '../../../packages/list';
```

Comment:
I don't think we should be doing this...I see the innerRef error, but this is indicating to us that there is an error with our types

Reply:
Ah I see, then let's fix this another PR!